daisy/step_create_images.go

```diff
@@ -116,12 +141,19 @@ func (ci *CreateImages) run(ctx context.Context, s *Step) DError {
 		ci.markCreatedInWorkflow()
 	}
 
+	if imageUsesAlphaFeatures(ci.ImagesAlpha) {
+		for _, i := range ci.ImagesAlpha {
+			wg.Add(1)
+			go createImage(i, i.OverWrite)
+		}
+	}
 	if imageUsesBetaFeatures(ci.ImagesBeta) {
```

Comment:
Beta could use Alpha features in which case `createImage` would be called twice (once for alpha, once for beta). This will not happen right now as isBeta returns `false` but it could happen in the future if we want to use a beta-only feature.

I think the best would be to have these `if`s separated with `else`. I.e.:

```
if alpha {
   createAlpha
} else if beta {
   createBeta
} else {
    create
}
```

Reply:
That makes sense, thanks for the feedback. Made this change in [90f4a2d](https://github.com/GoogleCloudPlatform/compute-image-tools/pull/1585/commits/90f4a2d7d536a8a77324428d4ba9a2a0757eaf8f)